modules/cli/src/main/scala/scala/cli/commands/ScalaCommand.scala

```diff
@@ -220,6 +220,22 @@ abstract class ScalaCommand[T](implicit myParser: Parser[T], help: Help[T])
         sys.exit(1)
     }
 
+  /** @param options
+    *   command-specific [[T]] options
+    * @return
+    *   by default [[sharedOptions]].logging, override to adjust.
+    */
+  def loggingOptions(options: T): Option[LoggingOptions] =
+    sharedOptions(options).map(_.logging)
+
+  /** @param options
+    *   command-specific [[T]] options
+    * @return
+    *   by default [[loggingOptions]].verbosity, override to adjust.
+    */
+  def verbosity(options: T): Option[Int] =
```

Comment:
I think `verbosity ` should always be set, why does this function return `Option`?
```suggestion
  def verbosity(options: T): Int =
```

Reply:
verbosity indeed could always be set... but then we couldn't make it dependent on `loggingOptions` (chain dependent on `sharedOptions`) and it would have to be overriden from the perspective of each sub-command.
which I guess is a lot of code duplication... but I could extract it to a trait, maybe. 🤔